src/07-even-more-loops.js

```diff
@@ -12,7 +12,11 @@
  *  shoutForLoop([ "A", "Very", "Happy", "Array" ]);
  *  //> [ "A!", "Very!", "Happy!", "Array!" ];
  */
-function shoutForLoop(array) {}
+function shoutForLoop(array) {
+  for(let i = 0; i < 10; i++){
+
```

Comment:
Take another shot here.  How do we make sure our loop traverses the entire length of the array